pkg/connectorbuilder/connectorbuilder.go

```diff
@@ -42,6 +47,7 @@ type builderImpl struct {
 	resourceBuilders       map[string]ResourceSyncer
 	resourceProvisioners   map[string]ResourceProvisioner
 	resourceProvisionersV2 map[string]ResourceProvisionerV2
+	eventFeed              EventProvider
```

Comment:
should this be `map[string]EventProvider`

Reply:
Hmm, maybe it is just `[]EventProvider`? I don't know that they are tied to a resource type specifically, but the idea of supporting multiple feeds makes sense to me.